src/components/ui/uploader.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+import React, { useCallback } from "react";
+import { useDropzone, type FileWithPath } from "react-dropzone";
+import { Button } from "./button";
+
+type uploadProps = {
+  uploadType: "avatar" | "companyLogo" | "incorporationDocument";
+};
+
+function Uploader({ uploadType }: uploadProps) {
+  const onDrop = useCallback((acceptedFiles: FileWithPath[]) => {
+    // Do something with the files
+    console.log(acceptedFiles, uploadType);
+  }, []);
+
+  const { acceptedFiles, getRootProps, getInputProps, open } = useDropzone({
+    // Disable click and keydown behavior
+    noClick: true,
+    noKeyboard: true,
+    onDrop,
+  });
+
+  //Shows a list of files that have been accepted
+  const files = acceptedFiles.map((file: FileWithPath) => (
+    <li key={file.path}>
+      {file.path} - {file.size} bytes
```

Comment:
Could you format bytes to kb, Mb, etc? 

Something like:

```ts
function getFileSizeSuffix(bytes: number): string {
  const suffixes = ['','K','M','G','T'];
  const magnitude = Math.floor(Math.log2(bytes) / 10);
  const suffix = suffixes[magnitude] + "B";
  return suffix;
}
```

eg. `getFileSizeSuffix(1024)` → `"KB"`